Project-Timer-frontend/src/environments/environment.ts

```diff
@@ -1,19 +1,11 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+import { firebase } from "./firebase.conf"
 
 export const environment = {
   production: false,
-  firebase: {
```

Comment:
pilnuj żeby klucze do api nie pojawiały się w repo, stwarza to poważne luki w bezpieczeństwie aplikacji